src/server/tests.rs

```diff
@@ -3082,74 +3083,86 @@ async fn test_code_action_import_insertion_full_path() {
 
 #[test]
 async fn compute_diagnostics_multi_file() {
-    let server = create_server();
+    async fn test_files(file_0: &str, file_1: &str) {
```

Comment:
Same test code as before. Just nested and run with different uris, to match our 2 scenarios.